maps/delta/outfits/guardsmen.dm

```diff
@@ -36,15 +31,20 @@ Begin Warhammer loadouts
 	flags = OUTFIT_HAS_BACKPACK|OUTFIT_NO_SURVIVAL_GEAR
 
 /decl/hierarchy/outfit/job/guardsman/krieg
-	name = OUTFIT_JOB_NAME("Krieger Guardsman")
+	name = OUTFIT_JOB_NAME("Krieg Guardsman")
 	uniform = /obj/item/clothing/under/rank/krieg_uniform
 	suit = /obj/item/clothing/suit/armor/krieger
 	back = /obj/item/storage/backpack/satchel/krieger
+	belt = /obj/item/device/flashlight/lantern
 	gloves = /obj/item/clothing/gloves/combat/krieg
 	shoes = /obj/item/clothing/shoes/jackboots/krieg
 	head = /obj/item/clothing/head/helmet/krieghelmet
 	mask = /obj/item/clothing/mask/gas/krieg
+	glasses = null
+	l_pocket = /obj/item/storage/box/ifak
+	r_pocket = null
 	suit_store = /obj/item/gun/energy/las/lasgun/luscius
+	neck = /obj/item/reagent_containers/food/drinks/canteen
```

Comment:
Remember that all undefined variables are inherited from the parent, so since the neck var was already defined in `/decl/hierarchy/outfit/job/guardsman` it's already defined in any `/decl/hierarchy/outfit/job/guardsman/subtype`. Of course if you're just adding redundancies for later contributors I understand. Nice catch on the glasses, I completely missed it.

Reply:
This is intentional, it's for other contributors to understand the system.